e2e/test/scenarios/embedding/embedding-reproductions.cy.spec.js

```diff
@@ -1082,24 +1154,64 @@ describe("issue 8490", () => {
     });
 
     cy.findByTestId("embed-frame").within(() => {
-      // X-axis labels: Jan 2023
-      cy.findByText("1월 2023").should("be.visible");
-
       // PDF export
       cy.findByText("PDF로 내보내기").should("be.visible");
-
       // Powered by
       cy.findByText("제공:").should("be.visible");
 
-      // Aggregation "count"
-      cy.findByText("카운트").should("be.visible");
+      cy.log("assert the line chart");
+      getDashboardCard(0).within(() => {
+        // X-axis labels: Jan 2024
+        cy.findByText("1월 2024").should("be.visible");
+        // Aggregation "count"
+        cy.findByText("카운트").should("be.visible");
+        // "Other" bar tooltip
+        otherSeriesChartPaths().first().realHover();
+      });
+    });
+
+    assertEChartsTooltip({
+      rows: [
+        {
+          name: "Gizmo",
+          value: "4",
+        },
+        {
+          name: "Widget",
+          value: "2",
+        },
+        {
+          name: "합계",
+          value: "6",
+        },
+      ],
+    });
+
+    cy.findByTestId("embed-frame").within(() => {
+      cy.log("assert the trend chart");
+      getDashboardCard(2).within(() => {
+        // N/A
+        cy.findByText("해당 없음").should("be.visible");
+        // (No data)
+        cy.findByText("(데이터 없음)").should("be.visible");
+      });
+
+      cy.log("assert the pie chart");
+      getDashboardCard(1).within(() => {
+        // Total
+        cy.findByText("합계").should("be.visible");
+        // Other
+        cy.findByTestId("chart-legend").findByText("기타").should("be.visible");
+      });
     });
 
     cy.log("test a static embedded question");
-    cy.get("@questionId").then(questionId => {
+
+    cy.log("assert the line chart");
+    cy.get("@lineChartQuestionId").then(lineChartQuestionId => {
```

Comment:
I don't want to repeat the same question test for each new question types, since they're basically the same, and visiting each question adds about 2 seconds on my machine, I could imagine it's a lot worse on CI with 2 more questions.